R/plot-observed-vs-simulated.R

```diff
@@ -115,6 +115,16 @@ plotObservedVsSimulated <- function(dataCombined,
     return(NULL)
   }
 
+  # In logarithmic scale, if any of the values are `0`, plotting will fail.
+  #
+  # To avoid this, just remove rows where any of the quantities are `0`s.
+  if (obsVsPredPlotConfiguration$yAxis$scale %in% c(tlf::Scaling$log, tlf::Scaling$ln)) {
```

Comment:
This was previously done during residuals calculation, but is only relevant for plotting.